.azure/templates/steps/system_test_general.yaml

```diff
@@ -87,6 +87,16 @@ jobs:
       displayName: "Create dir for Kafka binaries cache"
       condition: ne(variables['docker_tag'], 'latest')
 
+    - bash: |
```

Comment:
Should you give it some nice _display name_?

Reply:
I have `Build KafkaConnect's image with file-sink plugin` -> should it be `Build Connect's image with file-sink for STs`?

Reply:
Weird. GitHub does not seem to do the highlighting right, so I did not notice it. Is it well aligned? I guess it is if it works. So I think it is fine - I don't care that much whether it's one or the other wording. I just thought there was nothing because of the highlighting.

Reply:
Yeah it happens to me sometimes as well. It has same alignment as other `displayName` fields. Anyway thanks :)